<ui version="4.0" >
 <class>materials</class>
 <widget class="QWidget" name="materials" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Editar Material</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>311</width>
       <height>281</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>311</width>
       <height>281</height>
      </size>
     </property>
     <property name="title" >
      <string>Componente</string>
     </property>
     <widget class="QGroupBox" name="groupBox_8" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>40</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title" >
       <string>Ambient</string>
      </property>
      <widget class="QPushButton" name="BotonA" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>40</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title" >
       <string>Difus</string>
      </property>
      <widget class="QPushButton" name="BotonD" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_10" >
      <property name="geometry" >
       <rect>
        <x>90</x>
        <y>150</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title" >
       <string>Especular</string>
      </property>
      <widget class="QPushButton" name="BotonE" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QSlider" name="sliderSH" >
      <property name="geometry" >
       <rect>
        <x>100</x>
        <y>250</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum" >
       <number>128</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>250</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize" >
       <size>
        <width>71</width>
        <height>16</height>
       </size>
      </property>
      <property name="text" >
       <string>Shininess:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" >
     <item>
      <spacer name="horizontalSpacer_6" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonV" >
       <property name="text" >
        <string>&amp;Validar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonC" >
       <property name="text" >
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>17</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BotonC</sender>
   <signal>clicked()</signal>
   <receiver>materials</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>332</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel" >
     <x>357</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
